i18n/en.pot

```diff
@@ -1043,19 +1046,16 @@ msgid ""
 "Google. Please check the DHIS 2 documentation."
 msgstr ""
 
-msgid "Error"
-msgstr ""
-
 msgid "A connection to Google Earth Engine could not be established."
```

Comment:
Suggested text: "Cannot connect to Google Earth Engine."

Reply:
Fixed.